experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+
+> Is the `Cache-Control` header missing? Ensure you've installed the latest code in `aem-project/` to your publish instance. To do so with a single command, (from the `aem-project/` dir): `mvn -PautoInstallSinglePackagePublish clean install`
+
+Next, set the `enableTTL` flag to "1" by un-commenting the following line in your dispatcher.any configuration (on macOS: `/private/etc/apache2/conf/dispatcher.any`, around line 255):
+
+```
+/enableTTL "1"
+```
+
+You will need to restart Apache for this change to take effect:
+
+```
+sudo apachectl restart
+```
+
+Clear your dispatcher cache of any existing content. On macOS:
+
+```
+sudo rm -rf /Library/WebServer/docroot/publish/*
+```
+
+Finally, navigate to regular-page.html and observe the files which are written to the cache:
+
+http://aem-publish.local:8080/content/dispatchertester/us/en/regular-page.html
+
+Result:
+
+<img src="../img/enableTTL-ttl-file.png" width="900">
+
+Note how the last modified timestamp on `regular-page.html.ttl` is actually set _in the future_ by a minute - our `max.age`. Once the current time is _after_ the last modified timestamp on a resource's .ttl file, the file will be considered stale. 
+
+## In the logs
+
+The dispatcher.log output will differ slightly when using `enableTTL` vs. .stat file-based invalidation.
+
+### With `enableTTL`
+
+With `enableTTL` set and a corresponding .ttl file next to the cached resource in question, you will see log entries that match the following format.
+
+When the current time is BEFORE the last modified timestamp of a cached resource's .ttl file, it is considered VALID:
+
+```
+[..] cache file has not expired: /docroot/content/dispatchertester/us/en.html
+[..] cache-action for [/content/dispatchertester/us/en.html]: DELIVER
+```
+
+When the current time is AFTER the last modified timestamp of a cached resource's .ttl file, it is considered STALE:
+
+```
+[..] cache file has expired: /docroot/content/dispatchertester/us/en.html
+[..] cache-action for [/content/dispatchertester/us/en.html]: CREATE
+```
+
+### Without `enableTTL`
+
+When `enableTTL` is not set, or a .ttl file is missing for the resource in question, the logs will read differently. When the cached resource is newer than it's nearest .stat file, the cached file is considered VALID:
+
+```
+[..] cache file is newer than lastflush -> use cache [/docroot/content/dispatchertester/us/en.html]
+[..] cache-action for [/content/dispatchertester/us/en.html]: DELIVER
+```
+
+When it's older than it's nearest .stat file, the file is considered STALE:
+
+```
+[..] cache file is older than lastflush -> flush [/docroot/content/dispatchertester/us/en.html]
+[..] cache-action for [/content/dispatchertester/us/en.html]: CREATE
+```
+
+Ensure you are seeing the "cache file has not expired"/"cache file has expired" format in your local dispatcher logs (on macOS: `/private/var/log/apache2/dispatcher.log`) before proceeding.
```

Comment:
I think the location of dispatcher.log should be made clear at the top of this section.